cob_cartesian_controller/CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ add_library(cartesian_controller_utils src/cartesian_controller_utils.cpp)
 add_dependencies(cartesian_controller_utils ${catkin_EXPORTED_TARGETS})
 target_link_libraries(cartesian_controller_utils ${catkin_LIBRARIES})
 
-add_library(profile_generator src/trajectory_profile_generator/trajectory_profile_generator_lin.cpp src/trajectory_profile_generator/trajectory_profile_generator_circ.cpp)
+add_library(profile_generator src/trajectory_profile_generator/trajectory_profile_generator.cpp)
```

Comment:
As this file is quite long, I think splitting it up in two separate cpp files (maybe even a third one for the builder itself) would be desirable...